app/src/main/res/layout/activity_signup.xml

```diff
@@ -49,7 +51,9 @@
         app:layout_constraintBottom_toTopOf="@+id/textInfo2"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/textInfo1" />
+        app:layout_constraintTop_toBottomOf="@+id/textInfo1"
+        android:autofillHints=""
+        tools:ignore="LabelFor" />
```

Comment:
Probably same here, but the ids are also poorly chosen, unless they're placeholders, so can't know for sure if we should ignore LabelFor or not